src/_guides/language/language-tour.md

```diff
@@ -505,6 +505,14 @@ var y = 1.1;
 var exponents = 1.42e5;
```

Comment:
Not part of this change, but this doesn't cover things like `var exponent2 = 8e5` or `double d = 5;`, which are both doubles without a "decimal" (should that be "decimal point" above, or does "decimal" also mean a fractional digit?)

Reply:
Including `var exponent2 = 8e5;`, I think your point on `double d = 5;` is sufficiently covered by other sections.